packages/core/src/models/command-options.ts

```diff
@@ -259,6 +262,12 @@ export interface VersionCommandOption {
   /** Defaults to 'v', customize the tag prefix. To remove entirely, pass an empty string. */
   tagVersionPrefix?: string;
 
+  /** Do not update the project root lock file. */
+  noUpdateRootLockFile?: boolean;
```

Comment:
Whats the difference between `updateRootLockFile` and `packageLockfileOnly` ?

Reply:
`updateRootLockFile` is the flag to use when you want Lerna-Lite to update the lock file directly (what is currently in place for updating the npm lock file), while `packageLockfileOnly` would be the new flag in this PR. I'm considering keeping both approach but I agree that the flag `updateRootLockFile` is confusing and it could be renamed since it's not yet exposed in the version readme.

Reply:
Got it, thanks. Renaming would be good.